### ISSUE TITLE
.NET 5 WinForms Single File Publish is not working as expected

### ISSUE THREAD

**Issue by fatihyildizhan** [dotnet/wpf#3915]
It is creating net5.0-windows folder and many files to Release folder.

I upgraded from .NET Core 3.1 to .NET 5. It was working with .NET Core 3.1.

![net5](https://user-images.githubusercontent.com/602600/102463945-ca4aca00-405c-11eb-9fdd-78e01ba91144.png)

**Comment by thomasclaudiushuber** [comment]
Hey @fatihyildizhan , is this issue about WPF or about WinForms? The WinForms GitHub repo is here: [github.com/dotnet/winforms](https://github.com/dotnet/winforms)

Anyway, I assume there won't be a difference between WPF and WinForms regarding single-file publishing. So, if there's a problem with .NET 5, it might exist for WPF and WinForms. 

Could it be that you are looking at the build folder, not the publish folder? 

See also this:
[https://github.com/dotnet/designs/blob/main/accepted/2020/single-file/design.md#user-experience](https://github.com/dotnet/designs/blob/main/accepted/2020/single-file/design.md#user-experience)

**Comment by lindexi** [comment]
Duplicate of #3516 #3634 #3875 #3886?

**Comment by thomasclaudiushuber** [comment]
> Duplicate of #3516 #3634 #3875 #3886?

@lindexi Yes, I looked at the other issues too. But looks like @fatihyildizhan describes a problem that does not even produce a single file.

**Comment by uflilla** [comment]
Hi @fatihyildizhan,
Probably, you are just building the app. A single file is produced when the app is published. The Project config file must have the following settings.
```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net5.0</TargetFramework>
    <RootNamespace>Duplicate_Finder</RootNamespace>
    <IncludeAllContentForSelfExtract>true</IncludeAllContentForSelfExtract>
    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
    <PublishSingleFile>true</PublishSingleFile>
    <PublishReadyToRun>true</PublishReadyToRun>
  </PropertyGroup>

</Project>
```

Right Click on the folder in Solution Explorer and click Publish. The next steps are self-explanatory.

**Comment by IJlima** [comment]
> ```
>     <IncludeAllContentForSelfExtract>true</IncludeAllContentForSelfExtract>
> ```

This line is what made it work for me, I spend over 4 hours trying to find something, thanks man
